qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-PhoneNumberValidation/phoneNumberValidation.feature

```diff
@@ -0,0 +1,24 @@
+Feature:Phone Number validation
+
+  Background:
+    Given user goes to find patient record app
+   
+  @selenium
+  @phoneNumberValidation
+  Scenario: Phone Number validation
+    When user searches for patient using patient identifier
+    And user clicks on the first patient
+    Then system loads patient dashboard page
+    And user clicks on show contact link
+    And user clicks on edit contact info link
+    And user clicks on phone number button
```

Comment:
Where is the phone number button?

Reply:
This is the page https://qa-refapp.openmrs.org/openmrs/registrationapp/editSection.page?patientId=111&sectionId=contactInfo&appId=referenceapplication.registrationapp.registerPatient&returnUrl=%2Fopenmrs%2Fcoreapps%2Fclinicianfacing%2Fpatient.page%3FpatientId%3D5d563591-dbca-4f5b-a776-a344101b4449%26   . Unless you dont click on phone Number button. you wont be able to access phone Number input area. so there should be a first click on the tab.

Reply:
This is the page https://qa-refapp.openmrs.org/openmrs/registrationapp/editSection.page?patientId=111&sectionId=contactInfo&appId=referenceapplication.registrationapp.registerPatient&returnUrl=%2Fopenmrs%2Fcoreapps%2Fclinicianfacing%2Fpatient.page%3FpatientId%3D5d563591-dbca-4f5b-a776-a344101b4449%26   . Unless you dont click on phone Number button. you wont be able to access phone Number input area. so there should be a first click on the tab.

Reply:
Is that a button?

Reply:
changed it to Tab.